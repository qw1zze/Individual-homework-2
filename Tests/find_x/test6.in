1.3 1.8 0.000001