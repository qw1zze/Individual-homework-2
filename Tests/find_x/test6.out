1.726331468